src/store/databases.js

```diff
@@ -64,9 +67,43 @@ const actions = {
           toastr.error('Retrieving questions', 'Something went wrong')
         }
       })
+  },
+
+  createDatabase ({ commit }, data) {
+    const messageSubtitle = 'Creating database'
+    return axios.post(`${URL}/databases`, data)
+        .then(({ status, statusText, data }) => {
+          if (checkStatus(status, statusText)) {
+            toastr.success(messageSubtitle, 'Successful')
+            commit('addDatabase', data)
+          } else {
+            toastr.error(messageSubtitle, 'Something went wrong')
+          }
+
+          return data
+        })
+  },
+
+  createQuestion ({ commit }, data) {
+    const messageSubtitle = 'Creating question'
+    return axios.post(`${URL}/questions`, data)
+        .then(({ status, statusText, data }) => {
+          if (checkStatus(status, statusText)) {
+            toastr.success(messageSubtitle, 'Successful')
+            commit('addQuestion', data)
+          } else {
+            toastr.error(messageSubtitle, 'Something went wrong')
+          }
+
+          return data
+        })
   }
 }
 
+// statusText added for later use.
+function checkStatus(status, statusText) {
```

Comment:
👍  but rewrite `getQuestions` and `getDatabases` actions to use this helper as well.